test/Microsoft.Dnx.Testing.CommonUtils/Directory/Dir.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dir
+    {
+        public const string EmptyFile = "";
+        private readonly Dictionary<string, object> _nodes = new Dictionary<string, object>();
+
+        public Dir(string rootPath): this()
+        {
+            Load(rootPath);
+        }
+
+        public Dir()
+        {
+            ReadFile = fileInfo => File.ReadAllText(fileInfo.FullName);
+        }
+
+        public Dir(Dir parent)
+        {
+            ReadFile = parent.ReadFile;
+        }
+
+        public Func<FileInfo, object> ReadFile;
```

Comment:
My guess is that Wei had this to allow for flexibility in terms of file read behaviour. However, I haven't seen any scenarios where this is required yet. For now I'll remove it and maybe if there is a need for it in the future I'll add it back in.